docs/datagatherers/gke.md

```diff
@@ -101,8 +101,11 @@ resource "google_project_iam_member" "preflight_agent_cluster_viewer" {
 # if using workload identity in GKE, use the following binding to allow the
 # agent to use the service account
 resource "google_project_iam_binding" "preflight_agent_workload_identity" {
```

Comment:
Hey, the resource should also be `google_service_account_iam_binding` since we're trying to replicate a call like this:

```
gcloud iam service-accounts add-iam-policy-binding \
  --role roles/iam.workloadIdentityUser \
  --member "serviceAccount:cluster_project.svc.id.goog[k8s_namespace/ksa_name]" \
  gsa_name@gsa_project.iam.gserviceaccount.com
```

Thanks for doing this!

Reply:
https://www.terraform.io/docs/providers/google/r/google_service_account_iam.html#google_service_account_iam_binding

Reply:
Oh right! I missed that!